src/Data/StrMap.js

```diff
@@ -3,10 +3,12 @@
 
 // module Data.StrMap
 
+var hasOwnProperty = {}.hasOwnProperty;
```

Comment:
You can just delete this line. The global object already has this binding for you.